clusterctl/clusterdeployer/clusterclient.go

```diff
@@ -164,15 +167,31 @@ func (c *clusterClient) WaitForClusterV1alpha1Ready() error {
 
 func (c *clusterClient) kubectlApply(manifest string) error {
 	r := strings.NewReader(manifest)
-	cmd := exec.Command("kubectl", "apply", "--kubeconfig", c.kubeconfigFile, "-f", "-")
+	args := c.buildKubectlArgs("apply")
```

Comment:
Unless we are going to print args or do something else with it, just put the call inline below

`cmd := exec.Command("kubectl", c.buildKubectlArgs("apply")...)`